casper/src/test/scala/coop/rchain/casper/MultiParentCasperTestUtil.scala

```diff
@@ -67,7 +67,7 @@ object MultiParentCasperTestUtil {
         genesisPk = defaultGenesisPk,
         vaults = Vault(
           RevAddress.fromPublicKey(Secp256k1.toPublic(ConstructDeploy.defaultSec)).get,
```

Comment:
Thinking out loud: Should that be ConstructDeploy.defaultPub?

Reply:
I'll fix that in the next PR, or the rebase after my rename of that file are going to kill me :)